### PR TITLE
app created

### PR DESCRIPTION
**Pull request by Longlasting1805**
app created

### CODE REVIEW

File: blog/views.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from rest_framework.views import APIView
+from django.core.paginator import Paginator
+
+from .models import blog_model
+from .serializers import blog_serializer
+
+
+# Create your views here.
+
+class blog_view(APIView):
```
Comment (iamr0b0tx) [inline blog/views.py]: Class names are Pascal case

File: blog/serializers.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from .models import blog_model
+
+
+class blog_serializer(serializers.ModelSerializer):
```
Comment (iamr0b0tx) [inline blog/serializers.py]: All class names should be Pascal case

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: blog/serializers.py (hunk 2 above)
Comment (Longlasting1805) [reply blog/serializers.py]: Done sir

File: blog/views.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    def post(self, request):
+        serializer = BlogSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class BlogDetails(APIView):
```
Comment (iamr0b0tx) [inline blog/views.py:37]: Use viewset instead

**Review by iamr0b0tx — APPROVED** [review]

File: blog/views.py (hunk 3 above)
Comment (Longlasting1805) [reply blog/views.py:37]: Done

**Pull request merged**